src/dstack/_internal/core/models/configurations.py

```diff
@@ -193,8 +186,8 @@ class BaseRunConfiguration(CoreModel):
         Optional[RegistryAuth], Field(description="Credentials for pulling a private Docker image")
     ] = None
     python: Annotated[
-        Optional[PythonVersion],
```

Comment:
Why removing PythonVersion enum? I don't see much benefit allowing to specify a patch version.